### ISSUE TITLE
⌨ Vertical bars icon in notification dialog not read

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#10630]
## ⌨ Accessibility Issue

The icon representing the actions for a notification (specifically in directions) is not being read to screen readers and is confusing.

### WCAG 2.1 Criteria

I'm not actually sure there is specific criteria, maybe missing alt text for an informative image?

## 🦋 Expected Behaviour

Either the icon should be read out or the sentence should be reworded. I feel like the instructions do not make a lot of sense here for screen readers so maybe even omitting all of them? 🤔 

## 🕵️ Details

The test omits reading the icon tag (which sort of makes sense) so the sentence stops abruptly and sounds very unnatural for screen reader users. I don't think the instructions are very helpful and add more confusion that it solves for screen reader users.

## 📋 Steps to Reproduce

1. Start up any screen reader technology
2. Login and open then notification dialog
3. Have the instructions read out
4. Observe the unnatural way it reads past the icon

## 📸 Screenshot

![Screenshot 2024-06-11 085607](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/5d1afae5-e3c2-45a9-819e-d1090ea4896b)

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Instructions are read out naturally and make sense for all users

**Comment by gobyrne** [comment]
This text is getting removed in #10639
